modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusManager.scala

```diff
@@ -177,6 +190,28 @@ object ConsensusManager {
             .handleErrorWith(logger.error(_)(s"Error checking for consensus state update {key=${key.show}}"))
         }.void
 
+      private def getPeerCandidates(peers: List[Peer]): F[List[Peer]] = {
```

Comment:
`candidates` is already used in consensus in the context of peers that are going to be added as facilitators to the next consensus. Please use a different name to avoid confusion.

Reply:
Changed.